src/Resources/views/includes/layout/_content.html.twig

```diff
@@ -70,63 +70,6 @@
 
         {{ araise_table_render(content.table(view.data)) }}
 
-        {% set createUrl = content.createUrl(view.data) %}
-        {% if createUrl and (content.addVoterAttribute is null or is_granted(content.addVoterAttribute, view.data)) %}
-
-        <div
-            class="flex justify-end mt-3 mb-3 mx-3 {% if not isOnShow %}{{ content.option('attr')['class'] ?? '' }}{% endif %}"
-            {% if not isOnShow %}{{ attr|map((value, attr) => "#{attr}=\"#{value}\"")|join(' ')|raw }}{% endif %}
-            {{ stimulus_controller('araise/core-bundle/modal-form', { formUrl: createUrl }) }}
-        >
-            <button
-                type="button"
-                class="whatwedo-crud-button--action whatwedo-crud-button--small"
-                {{ stimulus_action('araise/core-bundle/modal-form', 'openModal') }}
-            >
-                {{ 'araise_crud.relation.add'|trans }}
-            </button>
-
-            <div
-                {{ stimulus_target('araise/core-bundle/modal-form', 'menu') }}
-                class="hidden whatwedo-crud-modal whatwedo-utility-modal"
-                data-transition-enter-from="opacity-0"
-                data-transition-enter-to="opacity-100"
-                data-transition-leave-from="opacity-100"
-                data-transition-leave-to="opacity-0"
-                role="dialog"
-                aria-modal="true"
-            >
-                <div class="whatwedo-crud-modal__overlay whatwedo-utility-modal__overlay">
-                    <div
-                        class="whatwedo-crud-modal__backdrop whatwedo-utility-modal__backdrop"
-                        {{ stimulus_action('araise/core-bundle/modal-form', 'close', 'click') }}
-                    ></div>
-
-                    {# This element is to trick the browser into centering the modal contents. #}
-                    <span class="hidden md:inline-block md:align-middle md:h-screen" aria-hidden="true">&#8203;</span>
-
-                    <div class="whatwedo-crud-modal__wrapper whatwedo-utility-modal__wrapper">
-                        <div class="whatwedo-utility-modal__wrapper-inner">
-                            <button
-                                type="button"
-                                class="whatwedo-crud-modal__close whatwedo-utility-modal__close"
-                                {{ stimulus_action('araise/core-bundle/modal-form', 'close') | stimulus_action('araise/core-bundle/modal-form', 'close', 'keydown.esc@window') }}
-                            >
-                                <span class="sr-only">Close</span>
-                                {{ bootstrap_icon('x', { class: 'inline w-8 h-8' }) }}
-                            </button>
-
-                            <div
-                                class="whatwedo-crud-modal__body w-full"
-                                {{ stimulus_target('araise/core-bundle/modal-form', 'modalBody') }}
-                            ></div>
```

Comment:
Nit: What's with that empty div? If this is a wrapper for something to load via JavaScript, we should add some fallback content. be it an empty or loading state.

Reply:
A proper state could be added there indeed.

For now we handle this in the Stimulus controller very simple.
`this.modalBodyTarget.innerHTML = 'Loading ...';`